src/appleseed.studio/mainwindow/project/projectbuilder.h

```diff
@@ -164,15 +165,11 @@ Entity* ProjectBuilder::edit_entity(
     return new_entity_ptr;
 }
 
-template <>
-inline renderer::Camera* ProjectBuilder::edit_entity(
-    renderer::Camera*                   old_entity,
-    renderer::Scene&                    parent,
-    const foundation::Dictionary&       values) const
+template <typename OldEntity, typename NewEntity>
```

Comment:
I would expect `OldEntity` and `NewEntity` to always be the same type; let's unify them in a single `Entity` param?